### PR TITLE
remove filter overrides for MacOS so tests can be run locally

### PR DESCRIPTION
**Pull request by pes10k**
Resolves brave/brave-browser#38005

### CODE REVIEW

**Review by bridiver — APPROVED** [review]

**Comment by bridiver** [comment]
this has no impact on CI/builds/etc... so merging without waiting for CI

**Comment by goodov** [comment]
why this is force-merged? This PR should have `CI/run-upstream-tests` label and CI must pass before merging it.

With force-merging this you most likely broke all PRs with `CI/run-upstream-tests` label enabled.

**Pull request merged**
